examples/tutorials/analysis-time/Variability_estimation.py

```diff
@@ -0,0 +1,193 @@
+"""
+Estimation of time variability in a lightcurve
+==============================================
+Compute a series of time variability significance estimators for a lightcurve.
+
+Prerequisites
+-------------
+
+Understanding the light curve estimator, please refer to the :doc:`light curve notebook </tutorials/analysis-time/light_curve`.
+For more in-depth explanation on the creation of smaller observations for exploring time variability, refer to :doc:`light curve notebook </tutorials/analysis-time/light_curve_flare`
+
+Context
+-------
+Frequently, after computing a lightcurve, we need to quantify its variability in the time domain, for example in the case of a flare, burst, decaying light curve in GRBs or heightened activity in general.
+
+There are many ways to define the significance of the variability.
+
+**Objective: Estimate the level time variability in a lightcurve through different methods.**
+
+Proposed approach
+-------------
+We will start by reading the pre-computed light curve for PKS 2155-304 that is stored in `$GAMMAPY_DATA` 
+To learn how to compute such an object, see the :doc:`light curve tutorial </tutorials/analysis-time/light_curve_flare`
+
+This tutorial will demonstrate how to compute different estimates which measure the significance of variability. These estimators range from basic ones that calculate the peak-to-trough variation, to more complex ones like fractional excess and point-to-point fractional variance, which consider the entire light curve. We also show an approach which utilises the change points inn Bayesian blocks as indicators of variability.
+"""
+######################################################################
+# Setup
+# ----------------------------------------------
+# As usual, we’ll start with some general imports…
+
+
+import logging
+import numpy as np
+import astropy.units as u
+from astropy.stats import bayesian_blocks
+from astropy.time import Time
+import matplotlib.pyplot as plt
+from gammapy.estimators import LightCurveEstimator
+from gammapy.estimators.utils import (
+    compute_lightcurve_doublingtime,
+    compute_lightcurve_fpp,
+    compute_lightcurve_fvar,
+)
+
+log = logging.getLogger(__name__)
+
+######################################################################
+# Load the light curve for the PKS 2155-304 flare directly from “$GAMMAPY_DATA/estimators”
+
+lc_1d = FluxPoints.read(
+    "$GAMMAPY_DATA/estimators/pks2155_hess_lc/pks2155_hess_lc.fits", format="lightcurve"
+)
+
+plt.figure(figsize=(8, 6))
+lc_1d.plot(marker="o")
+plt.show()
+
+######################################################################
+# # Methods to characterize variability
+# ---------------
+#  Amplitude maximum variation, relative variability amplitude and variability amplitude
+#
+# The amplitude maximum variation is the simplest method to define variability ([reference
```

Comment:
One question here is whether this should be part of an helper function. It is simple code, but can be tedious to repeat. 
If we had a `LightCurve` object, this could also become a method.